xbmc/video/VideoLibraryQueue.cpp

```diff
@@ -32,7 +32,7 @@
 using namespace std;
 
 CVideoLibraryQueue::CVideoLibraryQueue()
-  : CJobQueue(false, 1, CJob::PRIORITY_LOW_PAUSABLE),
```

Comment:
Why not keep the _PAUSABLE here? Seems like you want these paused during e.g. video playback (to prevent stuttering etc.), right?